src/audio_core/hle/adts.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#ifndef ADTS_ADT
+#define ADTS_ADT
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+
+struct ADTSData {
+    bool MPEG2;
+    uint8_t profile;
+    uint8_t channels;
+    uint8_t channel_idx;
+    uint8_t framecount;
+    uint8_t samplerate_idx;
+    uint32_t length;
+    uint32_t samplerate;
+};
+
+typedef struct ADTSData ADTSData;
```

Comment:
I think this isn't necessary in C++

Reply:
why is this resolved

Reply:
Sorry, I marked it as resolved since my local commits resolved the issue and I forgot to push them.